src/components/form/FormFieldLabel.tsx

```diff
@@ -51,29 +57,40 @@ interface Props {
  *
  * @internal
  */
-export const FormFieldLabel = forwardRef<HTMLLabelElement, Props>((props, ref) => {
-  const { children, isRequired = false, ...rest } = props;
+export const FormFieldLabel = forwardRef<HTMLLabelElement, FormFieldLabelProps>((props, ref) => {
+  const { children, isRequired = false, helpToggletip, ...rest } = props;
 
   if (!children) {
     return null;
   }
 
   return (
-    <LabelStyled ref={ref} {...rest}>
-      {children}
-      {isRequired && <RequiredIndicator aria-hidden="true">*</RequiredIndicator>}
-    </LabelStyled>
+    <LabelWrapper>
+      <LabelStyled ref={ref} {...rest}>
+        {children}
+        {isRequired && <RequiredIndicator aria-hidden="true">*</RequiredIndicator>}
+      </LabelStyled>
+      {helpToggletip && <ToggleTip side={PopoverSide.Right} {...helpToggletip} />}
+    </LabelWrapper>
```

Comment:
[question] Do we not want to make the placement (side) configurable?

Reply:
It's already overridable through `helpToggletipProps`